templates/freeradius/sqlcounter.j2

```diff
@@ -0,0 +1,65 @@
+# The dailycounter is included by default in the freeradius conf
+
+sqlcounter dailycounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
```

Comment:
New problem, tests failing because ` ${modules.sql.dialect}` is not found.
I tried `dialect = "{{ freeradius_sql.dialect }}"` and that worked. Not sure why the value isn't found in `sqlcounter` now but should I make this change?

Reply:
Please see https://github.com/openwisp/ansible-openwisp2/pull/223/files#r573134118